### ISSUE TITLE
训练不收敛，高达1左右

### ISSUE THREAD

**Issue by HueCheng** [SHUCV/digit#15]
[I1121 16:48:06.706434 27104 solver.cpp:244]     Train net output #0: loss = 0.871819 (* 1 = 0.871819 loss)
[I1121 16:48:06.706444 27104 sgd_solver.cpp:106] Iteration 79980, lr = 3.05176e-12
[I1121 16:48:08.158608 27104 solver.cpp:454] Snapshotting to binary proto file fcn11_full_iter_80000.caffemodel
[I1121 16:48:08.185741 27104 sgd_solver.cpp:273] Snapshotting solver state to binary proto file fcn11_full_iter_80000.solverstate
不知道是什么情况，直接下载github的caffe使用的，还将softmaxwithloss.cpp给覆盖了，依旧不收敛，一张图只能识别一个数字。。请问大概是什么问题？？

PS：下了链接里的caffemoel跑test，效果还行，但是热度图整体向右下方偏移，导致右边第一个数字都无法识别出来，请问又是啥情况==

**Comment by lin65505578** [comment]
同样的问题。
